BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -282,6 +282,7 @@
   gPlatformModuleTokenSpaceGuid.PcdFuncCpuInitHook   | 0x00000000
   gPlatformModuleTokenSpaceGuid.PcdFspsUpdPtr        | 0x00000000
   gPlatformModuleTokenSpaceGuid.PcdAcpiTableTemplatePtr | 0
+  gPlatformModuleTokenSpaceGuid.PcdFuncPciEnumHook   | 0x00000000

```

Comment:
maybe PcdPciEnumHookProc ?   It is easier to know it is a function pointer in this way.